### PR TITLE
LITE-18083 General Connect Flake8/pytest rules are applied

### PR DESCRIPTION
**Pull request by maxipavlovic**

### CODE REVIEW

File: tests/test_metas.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
-#  Copyright © 2020 Ingram Micro Inc. All rights reserved.
-
-import pytest
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
```
Comment (marcserrat) [inline tests/test_metas.py:1]: Copyright shall be 2020 - 2021, updaring last on each change of year afaik

File: tox.ini [hunk 2]
```diff
@@ -1,4 +1,9 @@
+[flake8]
+exclude = .idea,.git,venv*/,.eggs/,*.egg-info
+max-line-length = 100
+show-source = True
+
 [pytest]
 DJANGO_SETTINGS_MODULE = tests.dj.settings
-addopts = --random-order --cov-report=xml --cov=./dj_cqrs
+addopts = --cov-report=xml --cov=./dj_cqrs
```
Comment (net-free) [inline tox.ini:8]: Why test randomization is off?

File: dj_cqrs/controller/consumer.py [hunk 3]
```diff
@@ -32,7 +33,7 @@ def route_signal_to_replica_model(signal_type, cqrs_id, instance_data, previous_
     :param dict instance_data: Master model data.
     """
     if signal_type not in (SignalType.DELETE, SignalType.SAVE, SignalType.SYNC):
-        logger.error('Bad signal type "{}" for CQRS_ID "{}".'.format(signal_type, cqrs_id))
+        logger.error('Bad signal type "{0}" for CQRS_ID "{1}".'.format(signal_type, cqrs_id))
```
Comment (net-free) [inline dj_cqrs/controller/consumer.py:36]: Required index parameters don't make code clear and doest help (we still get F524 on missing parameters).
Maybe it's better ignore P101?

**Review by net-free — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 2 above)
Comment (maxipavlovic) [reply tox.ini:8]: It's still on, another plugin is used `pytest-randomly`

File: dj_cqrs/controller/consumer.py (hunk 3 above)
Comment (maxipavlovic) [reply dj_cqrs/controller/consumer.py:36]: I consider them helpful, so will leave as is.

**Review by d3rky — APPROVED** [review]

**Review by net-free — APPROVED** [review]

File: tests/test_metas.py (hunk 1 above)
Comment (maxipavlovic) [reply tests/test_metas.py:1]: Will discuss and change in a separate task.

**Pull request merged**